### DISCUSSION TITLE
Multiple feature tracks for plotBrowserTrack

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#1634] [Feature Requests]
Any chance we could get the option pf plotting multiple feature tracks in the plotBrowserTrack function? E.g. if the features argument could take a list of GRanges object.

Thanks

**Comment by rcorces** [comment]
Already implemented. Within the past year, I clarified the documentation on this.
https://github.com/GreenleafLab/ArchR/blob/f6c0388bd37023400794c9ae8562ad69e3ba9fd7/R/ArchRBrowser.R#L649-L652
